sema/rocreator/rocreator.yml

```
@@ -0,0 +1,53 @@
+base:  !resolve {env.domain}/my-name
+sameas:
+  - doi:4886060.34/whatever
+
+uses: emobon-obsrvatory  # check if yaml has a built in extensiom mechanism
+ro:
+  version: 1.2-DRAFT
+  profile:  !resolve {env.domain}/observatory-profile/latest
```

YAMLlint (1.35.1)

[error] too many spaces after colon

(colons)